### PR TITLE
Add more sample clients for examples/local.

### PR DESCRIPTION
**Pull request by enisoc**
@michael-berlin 

These are alternative clients that show how to do the same thing as examples/local/client.py in other languages. This came about because a user got stuck on the fact that php/demo.php was incompatible with examples/local.

### CODE REVIEW

File: examples/local/client.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+		return
+	}
+	if err := tx.Commit(); err != nil {
+		fmt.Printf("commit failed: %v\n", err)
+		return
+	}
+
+	// Read it back from the master.
+	fmt.Println("Reading from master...")
+	rows, err := db.Query("SELECT id,msg FROM test_table")
```
Comment (michael-berlin) [inline examples/local/client.go]: Put a space after the comma?

id, msg

File: examples/local/client.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	connStr = fmt.Sprintf(`{"protocol": "grpc", "address": "%s", "keyspace": "%s", "shard": "%s", "tablet_type": "%s", "streaming": %v, "timeout": %d}`,
+		*server, keyspace, shard, "replica", false, timeout)
+	dbr, err := sql.Open("vitess", connStr)
+	if err != nil {
+		fmt.Printf("client error: %v\n", err)
+		return
+	}
+	defer dbr.Close()
+
+	rows, err = dbr.Query("SELECT id,msg FROM test_table")
```
Comment (michael-berlin) [inline examples/local/client.go]: id, msg

File: examples/local/client.go [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+func main() {
+	keyspace := "test_keyspace"
+	timeout := (10 * time.Second).Nanoseconds()
+	shard := "0"
+
+	flag.Parse()
+
+	// Connect to vtgate.
+	connStr := fmt.Sprintf(`{"protocol": "grpc", "address": "%s", "keyspace": "%s", "shard": "%s", "tablet_type": "%s", "streaming": %v, "timeout": %d}`,
```
Comment (michael-berlin) [inline examples/local/client.go]: lol, nice

You're already using "keyspace" and "shard". That was fast ;-)

**Comment by michael-berlin** [comment]
LGTM

**Comment by enisoc** [comment]
I addressed the comments, and also added the new clients to the local_example test, so Travis should fail if you forget to fix this as part of #1378. :)

**Comment by michael-berlin** [comment]
LGTM for additional changes.

Feel free to merge it :)

**Pull request merged**